### DISCUSSION TITLE
Compatibility with Ender 3 Pro w/4.2.7 Upgraded Board?

### DISCUSSION THREAD

**Discussion by cds333** [mriscoc/Ender3V2S1#238] [Q&A] [answered]
Thinking of putting [this ](https://www.amazon.com/Creality-Firmware-Installation-Complete-Mounting/dp/B08L12ZX23)into my Ender 3 Pro...

Any reason the Ender3V2-422-BLT firmware would not work? Or is it for the 4.2.2 exclusively?

**Comment by mriscoc** [comment, marked as answer]
There are firmware versions for 4.2.7 boards, for example: https://github.com/mriscoc/Ender3V2S1/releases/download/20220716/Ender3V2-427-BLTUBL-20220716.bin

**Reply by cds333** [reply]
Ah! Thank you very much! I am an idiot thus I missed this line : 

"More configuration files at: https://github.com/mriscoc/Special_Configurations"

Much appreciated!
